modules/apps/27-interchain-accounts/host/keeper/keeper.go

```diff
@@ -261,3 +273,41 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	bz := k.cdc.MustMarshal(&params)
 	store.Set([]byte(types.ParamsKey), bz)
 }
+
+func newModuleQuerySafeWhitelist() []string {
```

Comment:
Even though the function is unexported, maybe nice to add a godoc?

Reply:
done